make/compiler/Makefile.clang-included

```diff
@@ -15,4 +15,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+ifneq (,$(findstring cray-prgenv,$(CHPL_ORIG_TARGET_COMPILER)))
```

Comment:
I think this is probably safe, but it might be a little more safe to do a "starts with" comparison. E.g.

```makefile
ifneq (, $(filter cray-prgenv-%,$(CHPL_ORIG_TARGET_COMPILER)))
```